### PR TITLE
started to reorganize style.css #14661

### PR DESCRIPTION
**Pull request by SleepyBearIV**
check visual elements in  resulted.php, sectioned.php, courseed.php and codeviewer.php  nothing should be missing, i have only changed places of the code

### CODE REVIEW

File: Shared/css/style.css [hunk 1]
```diff
@@ -2090,6 +2090,111 @@ Dialog Box END
...
+  margin-top: 10px;
+  line-height: 0.5em;
+  color: var(--color-text-header);
+  width: 100%;
+}
+
+.loginBoxheader div,
+.logoutBoxheader div {
+  padding: 4px 20px;
+  /*position: absolute;
```
Comment (a22danwe) [inline Shared/css/style.css:2185]: remove if not used

File: Shared/css/style.css [hunk 2]
```diff
@@ -2090,6 +2090,111 @@ Dialog Box END
 /* --------------===============################================-------------- *
  *                                    Generic functions                            *
  * --------------================################================--------------*/
+.loginBox .inputwrapper {
+  clear: both;
+  width: 100%;
+  padding: 3px 0 3px 0;
+  /*height: 30px;*/
```
Comment (a22danwe) [inline Shared/css/style.css:2097]: if not used, remove commented out code.

File: Shared/css/style.css [hunk 3]
```diff
@@ -2090,6 +2090,111 @@ Dialog Box END
...
+  line-height: 30px;
+}
+
+.inputwrapper #startdate, .inputwrapper #enddate{
```
Comment (a22danwe) [inline Shared/css/style.css:2101]: add space after #enddate to be consistent

File: Shared/css/style.css [hunk 4]
```diff
@@ -2090,6 +2090,111 @@ Dialog Box END
...
+}
+
+.loginBox .inputwrapperSmall input,
+.loginBox .inputwrapperSmall select {
+  float: right;
+  width: 50px;
+}
+
+.loginBox .inputwrapperSmall input[type="checkbox"] {
+  /* margin-right: 200px; */
```
Comment (a22danwe) [inline Shared/css/style.css:2130]: same here, remove if not used.

File: Shared/css/style.css [hunk 5]
```diff
@@ -2162,6 +2267,20 @@ input.inactive-button {
...
 }
 
+#before,
+#after {
+  /*width: 226px;*/
+  width: 100%;
+}
+
+#playlink {
+  /*width: 216px;*/
```
Comment (a22danwe) [inline Shared/css/style.css:2277]: remove

File: Shared/css/style.css [hunk 6]
```diff
@@ -2162,6 +2267,20 @@ input.inactive-button {
   width: 440px;
 }
 
+#before,
+#after {
+  /*width: 226px;*/
```
Comment (a22danwe) [inline Shared/css/style.css]: remove dead code

File: Shared/css/style.css [hunk 7]
```diff
@@ -2283,125 +2277,146 @@ input.inactive-button {
...
+#editSection {
+  /*top: 50%;
+  left: 50%;
+  margin-top: -245px;
+  margin-left: -236px;*/
 }
 
-.loginBox .inputwrapperSmall span {
-  float: right;
+.statsdiv{
```
Comment (a22danwe) [inline Shared/css/style.css:2344]: space here for consistency

**Review by a22danwe — CHANGES_REQUESTED** [review]
looks good, minor changes for consistency and some dead code that needs to be removed if its not used

**Review by a22danwe — CHANGES_REQUESTED** [review]
perfecto!

**Review by a22danwe — APPROVED** [review]
perfecto!

**Comment by d22mikwe** [comment]
LGTM!

**Pull request merged**